While AyatanaAppIndicator3 is available on Tumbleweed,
it is not available on SLE15.

Index: virt-manager-5.1.0/virtManager/systray.py
===================================================================
--- virt-manager-5.1.0.orig/virtManager/systray.py
+++ virt-manager-5.1.0/virtManager/systray.py
@@ -6,7 +6,6 @@
 
 import os
 
-import gi
 from gi.repository import Gio
 from gi.repository import Gtk
 
@@ -18,11 +17,11 @@ from .baseclass import vmmGObject
 from .connmanager import vmmConnectionManager
 
 
-# pylint: disable=ungrouped-imports
-try:  # pragma: no cover
-    # pylint: disable=no-name-in-module
-    gi.require_version("AyatanaAppIndicator3", "0.1")
-    from gi.repository import AyatanaAppIndicator3 as AppIndicator3
+try:
+    # pylint: disable=ungrouped-imports
+    import gi
+    gi.require_version('AppIndicator3', '0.1')
+    from gi.repository import AppIndicator3
 except Exception:  # pragma: no cover
     AppIndicator3 = None
 
